README.md

```diff
@@ -107,7 +107,7 @@ Enable rulesets via the "extends" property of your `.eslintrc` configuration fil
 
 ### With TypeScript
 
-Add `@typescript-eslint/parser` to the "parser" filed in your `.eslintrc` configuration file.  
+Add `@typescript-eslint/parser` to the "parser" filed in your `.eslintrc` configuration file.
```

Comment:
Prettier accidentally-ish removed the hard line breaks from here, `immutable-data.md` (line 81) and `prefer-tacit.md` (line 61). I can add them back, but for clarity I would recommend to use [backslashes for hard line breaks](https://spec.commonmark.org/0.30/#hard-line-breaks) instead of trailing spaces.

Reply:
I can go either way with whether there's a line break there or not. I'll leave it up to you whether you want to add them back or not.
And yes, I agree that backslashes should be how line breaks are declared.

Reply:
I don't think hard line breaks are necessary in these three cases, so I'm not going to add them back. 👍